src/polyfills/details/_base.scss

```diff
@@ -18,29 +18,31 @@ summary {
 }
 
 details {
-	border: 1px solid #ddd;
-	border-radius: 4px;
-
-	summary {
+	.details {
```

Comment:
Wouldn't this need an <code>&.details</code> instead? Otherwise this only applies to elements with the "details" classname inside of a details element.